.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-EMC/fv3atm
+  url = https://github.com/noaa-emc/fv3atm
```

Comment:
I don't think case matters for git urls, but we should stay consistent with how the url shows it and the rest of the file. Please make this NOAA-EMC instead of noaa-emc

Reply:
Oops. Fixed. Tahnks